apps/storybook/src/Utilities.stories.mdx

```diff
@@ -206,53 +206,30 @@ useCanvasEvents({ onPointerDown: handlePointerDown }); // also supported: `onPoi
 Compute and update a local React state on every React Three Fiber frame. Useful to re-render React components when the user pans/zooms.
 The hook accepts a callback, which:
 
-- receives the `camera` object and the current `VisCanvasContext` value;
-- computes and returns a state based on the camera's current scale and position (typically using a combination of
-  [camera-based](https://h5web-docs.panosc.eu/?path=/story/utilities--page#camera-based-utilities) and
-  [context](https://h5web-docs.panosc.eu/?path=/story/context--page) utilities).
+- receives the `camera` object;
+- should compute and return a state based on the camera's current scale and position (typically using camera-based [context](https://h5web-docs.panosc.eu/?path=/story/context--page) utilities).
 
 ```ts
-const { dataToHtml } = useVisCanvasContext();
-
-const htmlPt = useCameraState(
-  (camera, context) => dataToHtml(camera, new Vector3(x, y)),
-  [x, y, dataToHtml]
-);
-```
+useCameraState<T>(
+  factory: (camera: Camera) => T,
+  equalityFn: (prev: T, next: T) => boolean
+): T
```

Comment:
No more dependencies array since I'm using `useSyncedRef` to wrap the callback (instead of `useCallback`). This simplifies things a lot for consumers since they no longer have to configure ESLint.